### ISSUE TITLE
sizeConfig mediaQuery values min-width & max-width are acting like they are inverted

### ISSUE THREAD

**Issue by jsnellbaker** [prebid/Prebid.js#1952]
## Type of issue
Bug

## Description
When testing with the 1.0 sizeConfig functionality, the min-width and max-width values for the mediaQuery param don't seem to be operating like their name describes; they seem to be flipped.  

When I have a screen set to a value below the min-width (say 1200px), my bid request is rejected; but when I have the screen to a value above the min-width, my bid request is accepted and I see the bid on my test page.  The same behavior happens for the max-width setting, but with the opposite logic.

## Steps to reproduce
Load the attached test page.

## Test page
[hello_world_sizeConfig.html.zip](https://github.com/prebid/Prebid.js/files/1553266/hello_world_sizeConfig.html.zip)

**Comment by jaiminpanchal27** [comment]
I got confused too but its correct. 
Definition of min-width - apply these styles to all screens that are greater than whatever value.
Try this test page
https://jsfiddle.net/9wj3vm3e/2/

**Comment by mkendall07** [comment]
@jsnellbaker 
Your test doesn't replicate for me. Can you verify that it's broken on your end?

**Comment by jsnellbaker** [comment]
I was able to trace the issue I was having.  

I was actually testing with my dev branch instead of master and I was supplying the mediaTypes.banner.sizes attribute as `[300x250]` instead of `[[300x250]]`.  This caused a certain filter statement in the sizeMapping.js file to improperly assign a valid size to a var, which caused the rest of the code to assume there were no matching sizes.

It was working when I did the opposite of the mediaQuery rule (eg went below the min-width) because the code assumed there was no valid rule to enforce - so it just basically let everything pass through.

Closing this issue as invalid.
